guide/payments/receive.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: guide
+title: Receiving bitcoin
+description: An introduction to how receiving bitcoin works.
+nav_order: 3
+parent: Payments
+permalink: /guide/payments/receive/
+main_classes: -no-top-padding
+# image: /assets/images/guide/payments/receive/header.svg
+---
+
+# Receiving bitcoin
+
+<!--
+
+Editor's notes
+
+This page should cover what to do when receiving bitcoin, how to share and copy addresses etc.
+
+- What addresses to support (native segwit by default)
+- How to present QR codes (allow for specifying an amount)
+- Generating a new address 
+
+-->
+
+For someone to receive bitcoin, they must first provide the sender with a payment destination. They can do this by generating an address within their Bitcoin wallet application. A new address should be generated for every payment, as addresses should only be used once. This helps safeguards the receiver’s privacy.
+
+<!-- 
+Update glossary
+
+Each address that a Bitcoin application generates has an accompaniying private key. This private key allows the receiver to unlock the bitcoin that is sent to the address when the receiver chooses to spend it.
+
+Although addresses are not a problem to share publically, their accompanying private keys which typically generated from a single recovery phrase must be kept secure and private.
+-->
+
+## Preparing the payment request
+The simplest way to receive bitcoin is by generating and sharing just a Bitcoin address with the sender. Receivers can share the address by either showing it as a scannable QR Code or as plain text that can be shared via any regular communication tool like email, instant messaging, SMS, etc.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/simple-share.png"
+   retina = "/assets/images/guide/payments/receive/simple-share@2x.png"
+   alt-text = "Screen showing how receiver is able to share a bitcoin address as soon as they enter the request flow"
+   width = 250
+   height = 541
+   layout = "float-right-desktop"
+%}
+
+</div>
+
+There are multiple types of addresses. Sharing one that is incompatible with the sender's wallet application can prevent a successful payment. Read more about addresses such [address compatibility]({{ '/guide/glossary/#address-compatibility' | relative_url }}) issues in the glossary.
```

Comment:
```suggestion
There are several types of addresses. Sharing one that is incompatible with the sender's wallet application can prevent a successful payment. Read more about [address compatibility]({{ '/guide/glossary/#address-compatibility' | relative_url }}) issues in the glossary.
```

Reply:
i thought this component after the galley looked a bit messy. what do you think?

<img width="1440" alt="Screenshot 2021-05-12 at 15 23 55" src="https://user-images.githubusercontent.com/183140/117982249-1cef9000-b336-11eb-9603-b88348592a99.png">

Reply:
The text really is a tip though (or warning), so I'd prefer to make it read well.
The next paragraph is also a completely different subject. (Could have it's own sub-heading 'Additional payment details')

Reply:
nice ye ill add the heading "Inputting additional payment details" similar to how its phrased on the send page